### PR TITLE
Remove papercups chat and instead open slack link

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
Remove papercups chat widget and make it a button that opens our slack

image

### CODE REVIEW

File: airbyte-webapp/src/pages/routes.tsx [hunk 1]
```diff
@@ -155,6 +155,7 @@ export const Routing = () => {
         <SupportChat
           papercupsConfig={config.papercups}
           customerId={workspace.customerId}
+          onClick={() => (window as any).open(config.ui.slackLink, "_blank")}
```
Comment (Jamakase) [inline airbyte-webapp/src/pages/routes.tsx:158]: Won't adding `open` function for Window interface in `config.ts` fix this problem with typings?

File: airbyte-webapp/src/components/SupportChat/SupportChat.tsx [hunk 2]
```diff
@@ -26,18 +47,12 @@ const SupportChat: React.FC<IProps> = ({ papercupsConfig, customerId }) => {
   }, [customerId, papercupsConfig]);
 
   const theme = useContext(ThemeContext);
+  console.log(theme);
```
Comment (Jamakase) [inline airbyte-webapp/src/components/SupportChat/SupportChat.tsx:50]: Do we really want to log theme?

**Review by jamakase — APPROVED** [review]

**Review by sherifnada — APPROVED** [review]

File: airbyte-webapp/src/components/SupportChat/SupportChat.tsx (hunk 2 above)
Comment (michel-tricot) [reply airbyte-webapp/src/components/SupportChat/SupportChat.tsx]: Nope :)

File: airbyte-webapp/src/pages/routes.tsx (hunk 1 above)
Comment (michel-tricot) [reply airbyte-webapp/src/pages/routes.tsx]: Fixed

File: airbyte-webapp/src/components/SupportChat/SupportChat.tsx (hunk 2 above)
Comment (michel-tricot) [reply airbyte-webapp/src/components/SupportChat/SupportChat.tsx]: Fixed

**Review by cgardens — APPROVED** [review]

**Pull request merged**